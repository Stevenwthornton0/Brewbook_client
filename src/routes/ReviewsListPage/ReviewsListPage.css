.searchForm {
    display: flex;
    flex-direction: column;
    margin-right: auto;
    margin-left: auto;
    width: 100%;
}

.searchForm > h2 {
    font-size: 26px;
    font-weight: 400;
    margin-top: 100px;
    margin-left: auto;
    margin-right: auto;
}