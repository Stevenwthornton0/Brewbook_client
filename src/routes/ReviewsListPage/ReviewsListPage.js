import React, { Component } from 'react';
import SearchBar from '../../components/ReviewsList/SearchBar/SearchBar';
import './ReviewsListPage.css'

class ReviewsListPage extends Component {
    render() {
        return (
            <div className='searchForm'>
                <SearchBar />
            </div>
        )
    }
}

export default ReviewsListPage;