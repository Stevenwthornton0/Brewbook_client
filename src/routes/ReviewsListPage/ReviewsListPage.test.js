import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import ReviewsListPage from './ReviewsListPage';

Enzyme.configure({ adapter: new Adapter() })

describe(`ReviewsListPage smoke test`, () => {
    it(`renders without crashing`, () => {
        const wrapper = shallow(<ReviewsListPage />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})