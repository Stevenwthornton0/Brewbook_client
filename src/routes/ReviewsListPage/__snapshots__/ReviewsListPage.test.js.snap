// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsListPage smoke test renders without crashing 1`] = `
<div
  className="searchForm"
>
  <h2>
    Search for a brewery
  </h2>
  <SearchBar />
</div>
`;
